rosdep/base.yaml

```diff
@@ -6906,6 +6906,7 @@ nlohmann-json-dev:
     bionic: [nlohmann-json-dev]
     focal: [nlohmann-json3-dev]
     jammy: [nlohmann-json3-dev]
+    noble: [nlohmann-json3-dev]
```

Comment:
Could we elide this rather than add to it with:
```diff
+     '*': [nlohmann-json3-dev]
      bionic: [nlohmann-json-dev]
-     focal: [nlohmann-json-dev]
-     jammy: [nlohmann-json-dev]

```

Reply:
Yeah, that's fair.  I notice that `debian` is in much the same boat, so I'll fix them both.